openlibrary/core/search.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import absolute_import
+
+import urllib
+import urllib2
+import web
+import simplejson
+import six
+
+from infogami.utils import delegate
+
+from openlibrary.utils import dateutil
+from openlibrary.core.models import Edition
+from openlibrary.core.helpers import get_coverstore_url
+from openlibrary.core.helpers import bookreader_host
+from openlibrary.core.lending import config_http_request_timeout
+
+
+PRESET_QUERIES = {
+    'preset:thrillers': '(creator:"Clancy, Tom" OR creator:"King, Stephen" OR creator:"Clive Cussler" OR creator:("Cussler, Clive") OR creator:("Dean Koontz") OR creator:("Koontz, Dean") OR creator:("Higgins, Jack")) AND !publisher:"Pleasantville, N.Y. : Reader\'s Digest Association" AND languageSorter:"English"',
```

Comment:
This curation data shouldn't be buried in the code. Also, a more appropriate name would be american-thrillers.

Reply:
The main goal was to DRY usage of these queries (which are only related to Archive.org advancedsearch specifically)